fedmsg_migration_tools/cli.py

```diff
@@ -51,34 +55,49 @@ def cli(conf):
 @click.option('--topic', multiple=True)
 @click.option('--zmq-endpoint', multiple=True, help='A ZMQ socket to subscribe to')
 @click.option('--exchange')
-@click.option('--amqp-url')
-def zmq_to_amqp(amqp_url, exchange, zmq_endpoint, topic):
+def zmq_to_amqp(exchange, zmq_endpoint, topic):
     """Bridge ZeroMQ messages to an AMQP exchange."""
-    amqp_url = amqp_url or config.conf['amqp_url']
     topics = topic or config.conf['zmq_to_amqp']['topics']
     exchange = exchange or config.conf['zmq_to_amqp']['exchange']
     zmq_endpoints = zmq_endpoint or config.conf['zmq_to_amqp']['zmq_endpoints']
     topics = [t.encode('utf-8') for t in topics]
     try:
-        bridges_module.zmq_to_amqp(amqp_url, exchange, zmq_endpoints, topics)
+        bridges_module.zmq_to_amqp(exchange, zmq_endpoints, topics)
     except Exception:
         _log.exception('An unexpected error occurred, please file a bug report')
 
 
 @cli.command()
 @click.option('--publish-endpoint')
-@click.option('--bindings')
+@click.option('--exchange')
 @click.option('--queue-name')
-@click.option('--amqp-url')
-def amqp_to_zmq(amqp_url, queue_name, bindings, publish_endpoint):
+def amqp_to_zmq(queue_name, exchange, publish_endpoint):
     """Bridge an AMQP queue to a ZeroMQ PUB socket."""
-    amqp_url = amqp_url or config.conf['amqp_url']
-    queue_name = queue_name or config.conf['amqp_to_zmq']['queue_name']
-    bindings = bindings or config.conf['amqp_to_zmq']['bindings']
+    if exchange and queue_name:
```

Comment:
I fiddled with these settings a bit in fedora-messaging so I'll change them here :)